### DISCUSSION TITLE
geodesicsolutions.org SEO Updates

### DISCUSSION THREAD

**Discussion by vicos59** [geodesicsolutions-community/geocore-community#44] [Ideas]
All of the following is solely in the interest of helping past, present, and future users of the software discover that there is now a Community Edition and their site may live on after all...

I will do some tweaks to geodesicsolutions.org to add verbiage that will help people searching for terms related to the commercial versions and company find the new CE site. But first, I need to come up to speed on GitHub. I've always used 
SmartSVN on Windows.

I added geodesicsolutions.org to Google Search Console so we can further tweak the site and make sure it gets fully indexed.

@jonyo In order to make this functional we have to provide proof that we own the domain. Can you add the following string to the DNS as a TXT record? 

google-site-verification=PoexcLSkHZDtDW_sZAu61M3LRm1ZHueGa4RzbiDw0s8

I can also add you as a user to the Google Search Console if you have a Google account. Best to get that info to me offline if you are interested.

**Comment by jonyo** [comment]
> I will do some tweaks to geodesicsolutions.org to add verbiage that will help people searching for terms related to the commercial versions and company find the new CE site. But first, I need to come up to speed on GitHub. I've always used SmartSVN on Windows.

Great!  I added some info on the main (and currently only) page in the wiki here:

https://github.com/geodesicsolutions-community/geocore-community/wiki

Let me know if that helps!  Or if you find other really useful articles or videos that you found helped you out, feel free to add the links to that wiki page.

Coming from SVN  to GIT will be a learning curve, it's been so long since I've used SVN I don't even remember all the adjustments. I do remember how much better it seemed to work though.

> I added geodesicsolutions.org to Google Search Console so we can further tweak the site and make sure it gets fully indexed.

You know I set up analytics and all that but forgot to add it to search console 😰 - I just added it to my own.  That way I am the main owner on it since currently I'm managing the domain and hosting.  I'd be happy to add you, contact me off-board with your google account and I'll send you an invite if you want to help manage it.  Heck I'd be happy to invite you to the analytics as well if that will help you out.

I'm not shy to say what I don't know...  And I know "some" about SEO / analytics, and specifically know what is not good (basically does it pass the "smell test" and/or is it somehow useful to humans).  But I'm not that good at actual promotion like you are doing, so I welcome the help!

I'm sure it will really help the community grow, thanks!  Just let me know what else you need.

**Reply by vicos59** [reply]
msg'ed you privately on FB
